--- a/main.c
+++ b/main.c
@@ -606,7 +606,7 @@ int callback(long int inpos, long int outpos){
 	  if(aheadposition>=0 && aheadposition<graph && !(function==-1))
 	    buffer[aheadposition+19]='>';
 	}
-	fprintf(stderr,buffer);
+	fprintf(stderr,"%s",buffer);
        
 	if (logfile != NULL && function==-1) {
-	  fprintf(logfile,buffer+1);
+	  fprintf(logfile,"%s",buffer+1);
 	  fprintf(logfile,"\n\n");
 	  fflush(logfile);
 	}
