--- a/util/ulockmgr_server.c
+++ b/util/ulockmgr_server.c
@@ -124,7 +124,7 @@ static int receive_message(int sock, void *buf, size_t buflen, int *fdp,
 	return res;
 }
 
-static int closefrom(int minfd)
+static int closefrom_local(int minfd)
 {
 	DIR *dir = opendir("/proc/self/fd");
 	if (dir) {
@@ -384,7 +384,7 @@ int main(int argc, char *argv[])
 	signal(SIGTERM, sig_handler);
 	signal(SIGINT, sig_handler);
 
-	closefrom(5);
+	closefrom_local(5);
 
 	res = socketpair(AF_UNIX, SOCK_STREAM, 0, sock);
 	if (res == -1) {
